.storybook/preview-head.html

```diff
@@ -12,7 +12,8 @@
   }
 
   body {
-    margin: 0;
+    margin: 0 !important;
+    padding: 0 !important;
```

Comment:
Gross, I know, but it's just the storybook wrapper

Reply:
😈